sqlitebrowser/umbrel-app.yml

```
@@ -0,0 +1,46 @@
+manifestVersion: 1
+id: sqlitebrowser
+name: DB Browser for SQLite (DB4S)
```

Would call it like this so the name does not get cut off on the umbrel dashboard.
```suggestion
name: SQLite Browser
```